### DISCUSSION TITLE
[Feature]: GDPR compliant

### DISCUSSION THREAD

**Discussion by mittalyashu** [logchimp/logchimp#952] [Ideas]
Allow owners to set their LogChimp site with GDPR compliant, only after enabling this option will allow their customers to delete the data.

## Implementation 

In the Dashboard settings "GDPR Compliant" option can be added allowing their LogChimp site to follow EU GDPR rules.

## Questions 

* Should there be option for all users to export their data or only after enabling this option from Dashboard settings?
* Display on homepage about the LogChimp site being GDPR compliance?

Please share your feedback in the comments.

**Comment by jmdrawneek** [comment]
This can be a manual process to be compliant. So supplying a contact (email/form/phone number) to invoke the right to be forgotten + giving an admin the ability to remove a user is enough.

But this would obviously be nice as it would make it easier to maintain as an admin.
